### PR TITLE
OSSMDOC-215

### PR DESCRIPTION
**Pull request by neal-timpe**
Part of the effort to updating the security documentation to improve securing the control plane. This only covers updating the security module to include a bit more context.

https://issues.redhat.com/browse/OSSMDOC-215

Related to https://github.com/openshift/openshift-docs/pull/28594

### CODE REVIEW

File: service_mesh/v2x/ossm-security.adoc [hunk 1]
```diff
@@ -7,6 +7,15 @@ toc::[]
 
 If your service mesh application is constructed with a complex array of microservices, you can use {ProductName} to customize the security of the communication between those services. The infrastructure of {product-title} along with the traffic management features of {ProductShortName} can help you manage the complexity of your applications and provide service and identity security for microservices.
 
+.Before you begin
+
+If you have a project, add your project to the xref:../../service_mesh/v2x/installing-ossm.adoc#ossm-member-roll-modify_installing-ossm[`ServiceMeshMemberRoll` resource].
```
Comment (neal-timpe) [inline service_mesh/v2x/ossm-security.adoc:12]: @rcernich Jaime's issue, https://issues.redhat.com/browse/OSSMDOC-151, says that there are implications for Kube NetworkPolicy. But, your security guide https://docs.google.com/document/d/1YsyoZn9gxRP8P3Vwm5-SGxQ1XV9JuGApzgDADiK86KQ/edit#heading=h.a5017lgbe5a6, says that its not required once you create the SMMR. 

Is that the end of the story, or are there other issues if someone already has a NetworkPolicy object?

Comment (rcernich) [reply service_mesh/v2x/ossm-security.adoc:12]: I think the issue with NetworkPolicy is that once the namespace joins the mesh, access to services/pods in that namespace will not be accessible to callers outside the mesh.

File: service_mesh/v2x/ossm-security.adoc [hunk 2]
```diff
@@ -7,6 +7,20 @@ toc::[]
 
 If your service mesh application is constructed with a complex array of microservices, you can use {ProductName} to customize the security of the communication between those services. The infrastructure of {product-title} along with the traffic management features of {ProductShortName} can help you manage the complexity of your applications and provide service and identity security for microservices.
 
+.Before you begin
+
+If you have a project, add your project to the xref:../../service_mesh/v2x/installing-ossm.adoc#ossm-member-roll-modify_installing-ossm[`ServiceMeshMemberRoll` resource]. 
+
+[NOTE]
+====
+After you add the the namespace to the `ServiceMeshMemberRoll`, access to services or pods in that namespace are not accessible to callers outside the mesh.
```
Comment (rcernich) [inline service_mesh/v2x/ossm-security.adoc:16]: "are not" or "will not be?"

**Review by rcernich — APPROVED** [review]

**Comment by neal-timpe** [comment]
@yxun can you take a look? It's a small one.

**Review by yxun — APPROVED** [review]

File: service_mesh/v2x/ossm-security.adoc [hunk 3]
```diff
@@ -7,6 +7,20 @@ toc::[]
...
+
+[NOTE]
+====
+After you add the the namespace to the `ServiceMeshMemberRoll`, access to services or pods in that namespace will not be accessible to callers outside the mesh.
+====
+
+If you don't have a project, install the xref:../../service_mesh/v2x/prepare-to-deploy-applications-ossm.adoc#ossm-tutorial-bookinfo-overview_deploying-applications-ossm[Bookinfo sample application] and add it to the `ServiceMeshMemberRoll` resource. The sample application helps illustrate security concepts.
+
+[id="ossm-security-smcp_{context}"]
+== Securing your control plane
```
Comment (JStickler) [inline service_mesh/v2x/ossm-security.adoc:22]: Is this heading in the right place?  It looks odd in the output having a heading with no content and then immediately after another heading.

File: service_mesh/v2x/ossm-security.adoc [hunk 4]
```diff
@@ -7,6 +7,20 @@ toc::[]
 
 If your service mesh application is constructed with a complex array of microservices, you can use {ProductName} to customize the security of the communication between those services. The infrastructure of {product-title} along with the traffic management features of {ProductShortName} can help you manage the complexity of your applications and provide service and identity security for microservices.
 
+.Before you begin
+
+If you have a project, add your project to the xref:../../service_mesh/v2x/installing-ossm.adoc#ossm-member-roll-modify_installing-ossm[`ServiceMeshMemberRoll` resource]. 
+
+[NOTE]
+====
+After you add the the namespace to the `ServiceMeshMemberRoll`, access to services or pods in that namespace will not be accessible to callers outside the mesh.
```
Comment (JStickler) [inline service_mesh/v2x/ossm-security.adoc:16]: Should we say that this is by design?  Or something like, "After you add the the namespace to the `ServiceMeshMemberRoll`, services or pods in that namespace have access to other services and pods within the mesh, but will not be accessible to callers outside the mesh unless you configure it."

File: service_mesh/v2x/ossm-security.adoc (hunk 3 above)
Comment (neal-timpe) [reply service_mesh/v2x/ossm-security.adoc]: Good point. I think that was part of an earlier idea for a draft and I left it in.

File: service_mesh/v2x/ossm-security.adoc (hunk 4 above)
Comment (neal-timpe) [reply service_mesh/v2x/ossm-security.adoc:16]: I'm not sure adding by design adds to this note. I think the note acknowledges that it's by design and I think it's implied that you can configure it differently later. I think it points out that during this step, you'll see changes.

**Review by JStickler — APPROVED** [review]

**Pull request merged**